e2e/runner/inscription.go

```diff
@@ -0,0 +1,116 @@
+package runner
+
+import (
+	"bytes"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"github.com/pkg/errors"
+	"io"
+	"net/http"
+)
+
+type commitResponse struct {
+	Address string `json:"address"`
+}
+
+type revealResponse struct {
+	RawHex string `json:"rawHex"`
+}
+
+type revealRequest struct {
+	Txn     string `json:"txn"`
+	Idx     int    `json:"idx"`
+	Amount  int    `json:"amount"`
+	FeeRate int    `json:"feeRate"`
+	To      string `json:"to"`
+}
+
+// InscriptionBuilder is a util struct that help create inscription commit and reveal transactions
+type InscriptionBuilder struct {
+	sidecarUrl string
+	client     http.Client
+}
+
+func (r *InscriptionBuilder) GenerateCommitAddress(memo []byte) (string, error) {
```

Comment:
Is it possible to add comment to explain method

Reply:
added